### PR TITLE
For #15837

### LINKED ISSUE #15837: [OpenForce 2022] Optimize official documents
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Aim

Check the source file below, keep a space between Chinese and English words

Source file:
https://github.com/apache/shardingsphere/blob/master/docs/community/content/contribute/committer.cn.md

Official document link:
https://shardingsphere.apache.org/community/cn/contribute/committer/

### Example

`您可以报告bug，提交一个新的功能增强建议或者直接对以上内容提交改进补丁。`

After optimization：

`您可以报告 bug，提交一个新的功能增强建议或者直接对以上内容提交改进补丁。`

### PR DESCRIPTION
**Pull request by Swastyy**
Fixes #15837 

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

**Review by terrymanu — APPROVED** [review]

**Comment by menghaoranss** [comment]
@Swastyy there are conflicting files.

**Comment by terrymanu** [comment]
Please resolve conflicts

**Comment by Swastyy** [comment]
Yes @menghaoranss @terrymanu , I have corrected it.

File: docs/community/content/contribute/committer.cn.md [hunk 1]
```diff
@@ -22,17 +22,49 @@ ShardingSphere社区遵循[Apache Community’s process](http://community.apache
...
+ - 判断是否是长期 issue，如是，则标记为 pending；
+ - 判断 issue 类型，如：bug，enhancement，discussion 等；
+ - 标记 Milestone。
 
-4. Committer review PR时，可以进行squash and merge to master的操作, 如果有问题可以加上change request或者@相关人员协助处理。
+3. Committer 提交的 PR，需要根据PR类型和当前发布的周期标注 Label 和 Milestone。
+
+4. Committer review PR 时，可以进行 squash and merge to master 的操作, 如果有问题可以加上 change request 或者@相关人员协助处理。
+
+### 轮值顺序表
```
Comment (menghaoranss) [inline docs/community/content/contribute/committer.cn.md:45]: This part has been deleted in the previous version

Comment (Swastyy) [reply docs/community/content/contribute/committer.cn.md:43]: It is written two times, should I remove that line?

Comment (menghaoranss) [reply docs/community/content/contribute/committer.cn.md:43]: @Swastyy please refer to this pr https://github.com/apache/shardingsphere/pull/15838/files

File: docs/community/content/contribute/committer.cn.md [hunk 2]
```diff
@@ -22,17 +22,47 @@ ShardingSphere社区遵循[Apache Community’s process](http://community.apache
 
 ## 日常工作
 
-1. Committer需要每天查看社区待处理的Pull Request和Issue列表，负责问题的处理：
+1. 每周负责轮值的 Committer 需要每天查看社区待处理的Pull Request 和 Issue 列表，负责问题的处理：
```
Comment (yx9o) [inline docs/community/content/contribute/committer.cn.md:25]: Here is another.

Comment (Swastyy) [reply docs/community/content/contribute/committer.cn.md]: Hey @yx9o , I have redone it using the new merged branch, please review it once now.

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**